awx/main/management/commands/list_instances.py

```diff
@@ -46,7 +46,7 @@ def handle(self, *args, **options):
                 color = '\033[92m'
                 if x.capacity == 0 or x.enabled is False:
                     color = '\033[91m'
-                fmt = '\t' + color + '{0.hostname} capacity={0.capacity} version={1}'
+                fmt = '\t' + color + '{0.hostname} enabled={0.enabled} capacity={0.capacity} version={1}'
```

Comment:
I think we should make this much louder and only show a value if it's disabled:

```python
if x.capacity == 0:
    color = '\033[91m'
if x.enabled is False:
    color = '\033[90m[DISABLED] '
```

Reply:
That makes sense.  I've updated the PR.  Thanks!